src/com/facebook/buck/cxx/toolchain/WindowsPreprocessor.java

```diff
@@ -61,22 +59,13 @@ private static String prependIncludeFlag(String includeRoot) {
 
   @Override
   public Iterable<String> prefixHeaderArgs(Path prefixHeader) {
-    throw new UnsupportedOperationException("prefix header not supported by " + getClass());
-    // TODO(steveo) Should be easy to add support; will try @ later time,
-    // when I can test w/ Windows.
-    // "Forced Include": https://msdn.microsoft.com/en-us/library/8c5ztk84.aspx
-    // Space is allowed between flag and its pathname argument.
-    // E.g. something like this (space allowed between flag and its argument):
-    // return ImmutableList.of("/FI", resolver.getAbsolutePath(prefixHeader).toString());
+    return ImmutableList.of("/Yc", prefixHeader.toString());
   }
 
   @Override
   public Iterable<String> precompiledHeaderArgs(Path pchOutputPath) {
-    throw new UnsupportedOperationException("precompiled header not supported by " + getClass());
-    // TODO(steveo) Should be easy to add support; will try @ later time,
```

Comment:
Is there some unit tests we can possibly add for this?

Reply:
Now there are some =)